Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1381,6 +1381,20 @@ theorem lt_ord_succ_card (o : Ordinal) : o < (succ o.card).ord :=
   lt_ord.2 <| lt_succ _
 #align cardinal.lt_ord_succ_card Cardinal.lt_ord_succ_card
 
+theorem card_le_iff {o : Ordinal} {c : Cardinal} : o.card ≤ c ↔ o < (succ c).ord := by
+  rw [lt_ord, lt_succ_iff]
+
+/--
+A variation on `Cardinal.lt_ord` using `≤`: If `o` is no greater than the
+initial ordinal of cardinality `c`, then the cardinal of any of its realizations
+is no greater than `c`.
```

Comment:
This should probably just say
```suggestion
initial ordinal of cardinality `c`, then its cardinal is no greater than `c`.
```
so maybe delegate so we can fix this before merge.